src/app/analyze/stack/container-toggler/container-toggler.html

```diff
@@ -0,0 +1,4 @@
+<div class="container-toggler">
```

Comment:
Why is this class set with this name. Classes are used almost exclusively for styles but I can't find any styles. Sometimes we will use a class for identification for tests when the element is repeated and can't have an ID.